Changelog

```diff
@@ -1,4 +1,73 @@
+   - WEBUI: Implement 'Auto hide zero status filters' (skomerko) 62c5f41f39582036278e52a4343f797d72b61dcb
+   - WEBUI: Allow to filter torrent list by save path (skomerko) 04eb40376e7ddaeb51e4a621991b11b64d91e9e0
+   - WEBUI: Handle regex syntax error for torrent filtering (HamletDuFromage) 9a8572bd215eead276afc189c9a35d6d3baba433
+   - WEBUI: Add missing icons (skomerko) e069fbc37f9dfa7dcaae32238bdb4770b8587538
+   - WEBUI: Add link to 'List of alternative WebUI' wiki page in Options (Chocobo1) a91bac8aa02b8d562d2c5685f4db2bc4f6ff0a66
+   - WEBUI: Improve properties panel, torrent deletion dialog, filter list, subcategories, torrent deletion, statistics window (skomerko) fda797cb76526114941f595114ca3404246886cb 39dd415d43eaa9a2552273c667152a5c7e3e5d13 (5b7c9d5725d1e47dde39081e4fdadbc23cd40ca8 + 81509dfb6554f3cf5679a6dd560b37c54bc77a2c) 1b53fdf9ee40ca5472246c3bc073d47b7306588d 0ea35c54a347bbfddf97f0b61982416f27ee52b4 5e3161a3f96335ce906c0248ac888a3a55509c2a
+   - WEBUI: Allow to display only hostname in the Tracker column (skomerko) 9d0fa213bed6172d269afd7364769eca66f16631
+   - WEBUI: Show country/region name next to its flag when 'Resolve peer countries' is enabled (skomerko) f681e954c7d84d4add53cc6f8908337916efa94b
+   - WEBUI: Improve hash copy actions in context menu (skomerko) d19f7b12d950e23c739412d8a9d85da0cff7bb5a
+   - WEBUI: Support removing tracker from all torrents in WebUI/WebAPI (Thomas Piccirello) d2b2afad233ad41d4a4c98695bd6813e4e293417
+   - WEBUI: Display DHT information in the Status bar only when DHT is enabled (skomerko) 183c7c75b1949ca668d6b3d62d6ab2b8b6aeb42d
+   - WEBUI: Add 'Confirm torrent recheck' option (skomerko) c3224459db0ff4a058cb35cb73ab5eff76af3070
+   - WEBUI: Fix removing tracker URL with '|' character (Thomas Piccirello) 8a6207d3fc8a63d9989a49b5e7079e3a2dd1c45c
+   - WEBUI: Support managing web seeds (Thomas Piccirello) 81def39d8c358587da93f2e4272ac4673933ab12
+   - WEBUI: Add colors to log table rows (skomerko) 0e03e4f8a7febffe88f46662b13408ef809320bb
+   - WEBUI: Prevent text selection within tabs, menu items (skomerko) 6bfabad92fa4b91857537f10eda943d764c75c87
+   - WEBUI: Use correct text and background colors in RSS details view (skomerko) 2952480f377923536e080538ed9b4e669bd3786d
+   - WEBUI: Reduce padding in torrents table (skomerko) 4cc3fedf379c6a9a153e77b98ff776602b8cab9e
+   - WEBUI: Add WebAPI/WebUI for managing cookies (Thomas Piccirello) 6bbb7b71cdf5b3dfbe397e27187f85fa70dc399a
+   - WEBUI: Support updating RSS feed URL (Thomas Piccirello) 50acb670b0a3687a0be8162a6853f6c4feba0a44
+   - WEBUI: Add 'Engine' column to Search table (skomerko) 449ca96e28c56a217ec77d699125d6599f613ff0
+   - WEBUI: Add confirm dialog for Auto TMM (skomerko) 4ff0687b944a4b293ebf20d70a22b29ff4de8f24
+   - WEBUI: Add context menu to search tabs (skomerko) 87644441ad88b251913402ece9fd4b6b78f59ff0
+   - WEBUI: restore arrow keys table navigation (Hanabishi) dfe9daf25d677f77532db86262556653073a00b1
```

Comment:
I think this fixed something that was broken only in master, arrow navigation works in 5.0.2...

Reply:
@sledgehammer999 
I did not track the WebUI changes in detail, but nevertheless I noticed that they had regressions several times, which were hotfixed by subsequent commits. So you should carefully double-check all WebUI related commits to filter out such "hotfixes".

Reply:
OK. This version of the changelog was compiled semi heuristically by filter by commit title between the two branches and then manually deleting entries that were clearly internal.
I'll double check later to see if they just fixed stuff in master.